script/oss-check

```diff
@@ -195,15 +195,15 @@ def build(branch)
     perform("git fetch && git worktree add --detach #{dir} #{target}")
   end
 
+  Dir.chdir(dir) do
```

Comment:
Why was this moved? This will slow down OSSCheck by preventing incremental builds.

Reply:
I was getting build failures all the time IIRC

Reply:
unfortunately CI has deleted the build, but you can see that OSSCheck had failed before https://github.com/realm/SwiftLint/pull/2894/commits/956ea6f949d15f0fde264a0b92bbccb9d1c6ca20:  https://github.com/realm/SwiftLint/runs/251739677